web/src/pages/Home/counterCode.js

```diff
@@ -0,0 +1,12 @@
+export default `import { h, app } from "hyperapp"
+
+app({
+  init: 0,
+  view: state =>
+    h("div", {}, [
+      h("h1", {}, state),
+      h("button", { onclick: state => state - 1 }, "-"),
```

Comment:
The code uses "-" and "+", but the live example has buttons for "subtract" and "add".